sql/moz-fx-data-shared-prod/firefox_accounts_derived/fxa_delete_events_v2/query.sql

```diff
@@ -26,7 +26,9 @@ WHERE
   )
   AND DATE(`timestamp`) = @submission_date
   AND (
-    jsonPayload.type IN ('accountDeleted', 'accountDeleted.byRequest', 'DB.deleteAccount')
+    jsonPayload.type = 'DB.deleteAccount'
+    -- 'accountDeleted', 'accountDeleted.byRequest', 'accountDeleted.unverifiedSecondaryEmail', etc.
+    OR jsonPayload.type LIKE 'accountDeleted%'
```

Comment:
I'd prefer to match only the exact message types expected, but given that there's a major change to the FxA deletion logic in progress ([FXA-5813](https://mozilla-hub.atlassian.net/browse/FXA-5813)) I think matching on this prefix gives us a better chance of seamlessly handling that change.